trustee-on-GCP/trustee/containers/kbc.container

```diff
@@ -0,0 +1,13 @@
+[Unit]
```

Comment:
Do we need to replicate all the containers? Can we use the already existing quadlet definition?